### PR TITLE
Update settings from hw testing

### PR DESCRIPTION
**Pull request by AndyZe**

### CODE REVIEW

File: moco_control/src/moco_hw_interface.cpp [hunk 1]
```diff
@@ -95,9 +100,9 @@ bool MocoHWInterface::init() {
     }  // end for each joint
 
     registerInterface(&joint_state_interface_);     // From RobotHW base class.
-    registerInterface(&position_joint_interface_);  // From RobotHW base class.
     registerInterface(&velocity_joint_interface_);  // From RobotHW base class.
```
Comment (dvj) [inline moco_control/src/moco_hw_interface.cpp:104]: Does the `position_joint_interface_` need to go away? Alternatively asked: Should the `velocity_joint_interface_` also not be registered?

I'm a little unclear if the joint limits still function this way.

Comment (AndyZe) [reply moco_control/src/moco_hw_interface.cpp:104]: I'm also unclear on that, and can't test it without hardware

Comment (AndyZe) [reply moco_control/src/moco_hw_interface.cpp:104]: Make an issue and we'll look at it later?

Comment (AndyZe) [reply moco_control/src/moco_hw_interface.cpp:104]: I reverted this line and made an issue

**Pull request merged**